tests/interactive/test_app.py

```diff
@@ -81,17 +82,65 @@ async def test_input_and_buttons():
         }
         """
         )
+
+        # press clear input button
+        await pilot.click("#clear_input_button")
+
+        # assert that the curent_module and test_module's are structurally equivalent
         await pilot.pause()
-        assert (
-            app.input_text_area.text
-            == """
+        assert app.input_text_area.text == ""
+
+        # Testing a pass
+        app.input_text_area.insert(
+            """
         func.func @hello(%n : index) -> index {
           %two = arith.constant 2 : index
           %res = arith.muli %n, %two : index
           func.return %res : index
         }
         """
         )
-        await pilot.click("#clear_input_button")
+
+        # Select a pass
```

Comment:
This should be in a separate test. What exactly breaks when you split them out? Let's figure this out before merging, please let me know what you've tried and what breaks exactly

Reply:
ok that's the whole point of cloning them, let's make them not shared